Simulator/Simulator.sor.xml

```diff
@@ -8,7 +8,7 @@
 -->
 <csmSor:Simulator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:csmSor="http://www.cosmotech.com/csm_simulator/10.0">
   <csmSor:Name>Brewery</csmSor:Name>
-  <csmSor:Version><![CDATA[1.2.1]]></csmSor:Version>
+  <csmSor:Version><![CDATA[1.3.0-dev.41]]></csmSor:Version>
```

Comment:
Could you remove this change from the PR? We'll add a clean commit later to bump to 1.3.0 :slightly_smiling_face: 
Same for the simulator version in Solution.yaml file